spt/strafestuff.cpp

```diff
@@ -41,6 +41,9 @@ ConVar tas_strafe_version("tas_strafe_version",
                           FCVAR_TAS_RESET,
                           "Strafe version. For backwards compatibility with old scripts.");
 
+ConVar tas_strafe_afh_length("tas_strafe_afh_length", "0.0000000000000000001", FCVAR_TAS_RESET, "Magnitude of AFHs");
```

Comment:
Is this something you can actually input? Also, is this cvar needed? Maybe just hardcode the value

Reply:
yes and I figured I'd leave it in case there's a problem with the default value of the cvar not working for whatever reason in some scenario

Reply:
I had some instances where it would work for some jumps and not for others and I'm not really sure why

Reply:
Oh, okay. I mean, it is pretty small, so there's a good chance that it underflows to 0 on some operation.